### DISCUSSION TITLE
[Idea] Multiple expeditions

### DISCUSSION THREAD

**Discussion by web-cooking-factory** [PrestaShop/PrestaShop#31596] [Ideas]
### The problem

Hello,

it's my first discussion so sorry if it has already been discussed or if I'm doing that wrong.

One thing missing in Prestashop is the ability to set multiple expeditions for one order. It becomes tricky when you set a synchro with an ERP like Dolibarr or other that manages it.

### The solution

It would require small changes in database structure I think. In BO we should be able to choose products in the cart and set an expedition for them. It could be the same process as refund.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Closing as the team is working on it and it's coming to PrestaShop 9.1.
